src/components/Collaborative/utils/replaceHrefReferenceWithHash.ts

```diff
@@ -0,0 +1,14 @@
+import md5 from "md5";
+
+export default function replaceHrefReferenceWithHash(content) {
+    const regex = /<a[^>]*>([^<]*)<\/a>/g;
```

Comment:
suggestion (non-blocking): create the regex in the global context so it's compiled only once